ext/apm-seata-skywalking-plugin/config/agent.config

```diff
@@ -0,0 +1,117 @@
+
+# If true, skywalking agent will enable profile when user create a new profile task. Otherwise disable profile.
+# profile.active=${SW_AGENT_PROFILE_ACTIVE:true}
+
+# Parallel monitor segment count
+# profile.max_parallel=${SW_AGENT_PROFILE_MAX_PARALLEL:5}
+
+# Max monitor segment time(minutes), if current segment monitor time out of limit, then stop it.
+# profile.duration=${SW_AGENT_PROFILE_DURATION:10}
+
+# Max dump thread stack depth
+# profile.dump_max_stack_depth=${SW_AGENT_PROFILE_DUMP_MAX_STACK_DEPTH:500}
+
+# Snapshot transport to backend buffer size
+# profile.snapshot_transport_buffer_size=${SW_AGENT_PROFILE_SNAPSHOT_TRANSPORT_BUFFER_SIZE:50}
+
+# Backend service addresses.
+collector.backend_service=${SW_AGENT_COLLECTOR_BACKEND_SERVICES:127.0.0.1:11800}
+
+# Logging file_name
+logging.file_name=${SW_LOGGING_FILE_NAME:skywalking-api.log}
+
+# Logging level
+logging.level=${SW_LOGGING_LEVEL:INFO}
+
+# Logging dir
+# logging.dir=${SW_LOGGING_DIR:""}
+
+# Logging max_file_size, default: 300 * 1024 * 1024 = 314572800
+# logging.max_file_size=${SW_LOGGING_MAX_FILE_SIZE:314572800}
+
+# The max history log files. When rollover happened, if log files exceed this number,
+# then the oldest file will be delete. Negative or zero means off, by default.
+# logging.max_history_files=${SW_LOGGING_MAX_HISTORY_FILES:-1}
+
+# Listed exceptions would not be treated as an error. Because in some codes, the exception is being used as a way of controlling business flow.
+# Besides, the annotation named IgnoredException in the trace toolkit is another way to configure ignored exceptions.
+# statuscheck.ignored_exceptions=${SW_STATUSCHECK_IGNORED_EXCEPTIONS:}
+
+# The max recursive depth when checking the exception traced by the agent. Typically, we don't recommend setting this more than 10, which could cause a performance issue. Negative value and 0 would be ignored, which means all exceptions would make the span tagged in error status.
+# statuscheck.max_recursive_depth=${SW_STATUSCHECK_MAX_RECURSIVE_DEPTH:1}
+
+# Mount the specific folders of the plugins. Plugins in mounted folders would work.
+plugin.mount=${SW_MOUNT_FOLDERS:plugins,activations}
+
+# Exclude activated plugins
+# plugin.exclude_plugins=${SW_EXCLUDE_PLUGINS:}
+
+# If set to true, the parameters of the sql (typically java.sql.PreparedStatement) would be collected.
+plugin.jdbc.trace_sql_parameters=${SW_JDBC_TRACE_SQL_PARAMETERS:true}
+
+# Kafka producer configuration
+# plugin.kafka.bootstrap_servers=${SW_KAFKA_BOOTSTRAP_SERVERS:localhost:9092}
+# if you want to set namespace. please make sure the OAP server has set it in Kafka fetcher module
+# plugin.kafka.namespace=${SW_KAFKA_NAMESPACE:""}
+
+# Match spring bean with regex expression for classname
+# plugin.springannotation.classname_match_regex=${SW_SPRINGANNOTATION_CLASSNAME_MATCH_REGEX:}
+
+# If set to true, to identify the current node as Seata Server
+plugin.seata.server=${SW_SEATA_SERVER:true}
```

Comment:
the current config should be mark as unchangeable

Reply:
fix